### ISSUE TITLE
Can't send messages 

### ISSUE THREAD

**Issue by kaiyes** [copleykj/socialize-messaging#5]
I made a youtube video showing the issue I am facing. 
https://www.youtube.com/watch?v=5MUthfF2XRA

As soon as I use
`.addParticipant( " userId" )` , it creates 17 more entries in the participant db. Is that intended? Also what do I put in the

`sendMessage( "body", callback)` method as `callback` ?

On other hand, I think that if someone wants to add someone to a chat, 
he himself should be able to it. Right now I can only add another user to the conversation from his own console using

`.addParticipant( user )`

where he himself is the currentUser.

Is it because I am using auto publish yet that I am facing this issue? In friendship package, as soon as a user requests a friendship, a requesterId and a userId is generated on the fly & that can be used for simple notification by finding all the requests that have been made for this user by checking

`Meteor.requests.find({ userId })`

That was real convenient. I guess something like that will be great for messaging too!

**Comment by copleykj** [comment]
@kaiyes Ok, so a couple things.. With adding participants, the parameter needs to be an array of user objects. I'm guessing that the insecure package is still active on your app. Once you remove it, the package should give an access denied for that operation.. I really should be doing a type check in that function though and I'll see what I can do about adding that shortly.

Also adding participants can only be done by other users who are actively participating in the conversation so you'll need to run the `addParticipants` call from the user who creates the conversation initially.

With the call back, it's optional and I rarely ever use it. If you need to use it though, it's just forwarded to the insert method, so the same signature is required. 

The error you are seeing though is a failure on my part to call `api.use` for the `socialize:server-time` package. For now adding it manually should fix that issue.

**Comment by kaiyes** [comment]
@copleykj  thanks a lot for answering. I know you are super busy & this means a lot for me. 
For now, I have only two user. After I call `new Conversation.save()` from user1,  I could only add user2 as participant in that conversation by calling `addParticipant` from user2. I wasn't able to call it from user1. I mean I could call it from user1 but it didn't add any user. But as far as I understand  `you'll need to run the addParticipants call from the user who creates the conversation initially.   `  user1 should be able to add user2 as participant. 

`With adding participants, the parameter needs to be an array of user objects.`. But I could add a single participant by passing the userId as a string like `addParticipant("1234asdfiUserId")!! 


`For now adding it manually should fix that issue.` you mean by adding the package itself using `meteor add socialize:server-time` or by  finding the `api.use` in the socialize-messaging.js and adding it there manually?

**Comment by copleykj** [comment]
Ok, I've published a new version `0.4.3` which corrects a regression. This is tested to work as expected now. You should be able to use it as intended now.

```javascript
var con = new Conversation().save();
con.addParticipant(Meteor.users.findOne({username:"lynda"}));
```

**Comment by kaiyes** [comment]
Perfect. Works now. Thank you. That was super fast.
